Description: Do not build libraries in debug mode, either shared or static.
Origin: vendor
Forwarded: not-needed
Author: Cristian Greco <cristian@debian.org>

--- a/build/rules/compile
+++ b/build/rules/compile
@@ -13,8 +13,8 @@
         clean static_debug static_release shared_debug shared_release
 
 all: $(DEFAULT_TARGET)
-all_static: static_debug static_release
-all_shared: shared_debug shared_release
+all_static: static_release
+all_shared: shared_release
 all_debug: static_debug shared_debug
 all_release: static_release shared_release
 
